src/xenserver/opt/xensource/www/XCP-ng-index.html

```diff
@@ -286,6 +289,96 @@
       .support p {
         font-weight: 600;
       }
+
+      /*----------------------
+      Spinner
+    ----------------------*/
+    .lds-default {
+        margin-left:auto;
+        margin-right:auto;
+        position: relative;
+        width: 8em;
+        height: 8em;
+        font-size:3px;
+      }
+      .lds-default div {
+        position: absolute;
+        width: 0.6em;
+        height: 0.6em;
+        background: #fff;
+        border-radius: 50%;
+        animation: lds-default 1.2s linear infinite;
+      }
+      .lds-default div:nth-child(1) {
+        animation-delay: 0s;
+        top: 3.7em;
+        left: 6.6em;
+      }
+      .lds-default div:nth-child(2) {
+        animation-delay: -0.1s;
+        top: 2.2em;
+        left: 6.2em;
+      }
+      .lds-default div:nth-child(3) {
+        animation-delay: -0.2s;
+        top: 1.1em;
+        left: 5.2em;
+      }
+      .lds-default div:nth-child(4) {
+        animation-delay: -0.3s;
+        top: 0.7em;
+        left: 3.7em;
+      }
+      .lds-default div:nth-child(5) {
+        animation-delay: -0.4s;
+        top: 1.1em;
+        left: 2.2em;
+      }
+      .lds-default div:nth-child(6) {
+        animation-delay: -0.5s;
+        top: 2.2em;
+        left: 1.1em;
+      }
+      .lds-default div:nth-child(7) {
+        animation-delay: -0.6s;
+        top: 3.7em;
+        left: 0.7em;
+      }
+      .lds-default div:nth-child(8) {
+        animation-delay: -0.7s;
+        top: 5.2em;
+        left: 1.1em;
+      }
+      .lds-default div:nth-child(9) {
+        animation-delay: -0.8s;
+        top: 6.2em;
+        left: 2.2em;
+      }
+      .lds-default div:nth-child(10) {
+        animation-delay: -0.9s;
+        top: 6.6em;
+        left: 3.7em;
+      }
+.lds-default div:nth-child(11) {
+  animation-delay: -1s;
+  top: 6.2em;
+  left: 5.2em;
+}
+.lds-default div:nth-child(12) {
+  animation-delay: -1.1s;
+  top: 5.2em;
+  left: 6.2em;
+}
+@keyframes lds-default {
+  0%, 20%, 80%, 100% {
+    transform: scale(1);
+  }
+  50% {
+    transform: scale(1.5);
+  }
+}
```

Comment:
it's new code

Reply:
Icons are from icones.js.org, I selected icons thare are completly open source, I also create some from bitmaps icons and modify them all to have a result near from FA.

Reply:
> it's new code

What I meant is simply that is was not indented the same way as the lines just before it. Now it's fixed thanks to the prettier format you ran.

Reply:
I always work with prettier with vanilla config on file save, but at the first time I didn't want to make a lot of change in the file to simplify the PR review.

Reply:
What I wanted initially was just that you would indent the lines you added the same way as the lines that where just before them.

I'm not against running prettier on the whole file, but ideally, after the review, I'd like the final git history to look like this:
* www: replace icons, drop unused external deps, remove unused CSS
* www: prettify XCP-ng-index.html

Reply:
Ok but how to make this possible ? I made a commit again ? I modify the PR description ?

Reply:
This will require a rebase of the commits, and squashing some together, and moving the prettifying commit to the end (but /!\ the commit named "prettier format" is not in fact the true commit which does the prettify change, and I think there may be other changes which are not just style, hidden in the commit).

However, the way the history currently is, I'm afraid there will be conflicts in the operation, so it might be simpler to rework the PR without prettification at all, and only do the prettification when everything else is OK.

Reply:
Actually, the title addition should also remain a separate commit, because it doesn't address the same set of changes, so the target commit structure would be:

* www: replace icons, drop unused external deps, remove unused CSS
* www: prettify XCP-ng-index.html
* www: add a title to the web page

Reply:
The command would be `git rebase -i master` and then plan the appropriate actions. However, if you have never done it, this will be challenging, because at the same time you will have lots of conflicts to solve if you attempt to move most changes before the prettifying. Given this, it might be easier to branch from your first commit and do the rest of the changes again, without the prettifying (but with the localized indentation fix), submit it as a new PR which references this one, and wait for the PR to be approved before. Then we can think of the prettifying as a new PR, or just decide not to do it.

Reply:
I prefer not to do it but I need I think